EmpleoDotNet/Scripts/Empleo.js

```diff
@@ -0,0 +1,38 @@
+﻿function UpdateTotalCount() {    
+    var total = parseInt($('.total .value').html());
+
+    $('.total .value').html(total - 1);
+}
+
+function UpdateApprovalStatus(jobId, isApproved) {
+    var self = $(this);
```

Comment:
Si eso lo sé. Lo que no sé es como lograr el separation of concerns en javascript. Si fuera C# tuviera clases (o layers si es necesario) para cada cosa y se comunicaran apropiadamente. 
¿Cómo puedo lograr lo mismo aquí?